functions/palindrome_number.py

```diff
@@ -1,2 +1,14 @@
 def palindrome_number(x):
-    pass
+    temp = x
+    reverse = 0
+    while(x>0):
+        dig = x%10
+        reverse = reverse*10 + dig
+        x = x//10
+
+    if temp == reverse:
```

Comment:
maybe just reverse the input and return it for shorter code - go google it online